include/powsybl/iidm/converter/Properties.hpp

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef POWSYBL_IIDM_CONVERTER_PROPERTIES_HPP
+#define POWSYBL_IIDM_CONVERTER_PROPERTIES_HPP
+
+#include <map>
+#include <string>
+
+namespace powsybl {
+
+namespace iidm {
+
+namespace converter {
+
+class Properties {
```

Comment:
I think we should use the same properties class in IIDM and in the converter. Consider to have only one class, and move this class in `stdcxx` namespace